### PR TITLE
Added recipe for python3-openid

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

**Comment by pmlandwehr** [comment]
Builds fine locally, so I'm kind of flummoxed.

**Comment by pmlandwehr** [comment]
Must admit I'm a bit flummoxed by this...

File: recipes/python3-openid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - defusedxml
```
Comment (jakirkham) [inline recipes/python3-openid/meta.yaml]: The issue is some of the builds for this failed and did not upload anything.

**Comment by pmlandwehr** [comment]
ok, close/reopening...

File: recipes/python3-openid/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/python3-openid/meta.yaml]: FWIW they seem to be up now.

**Comment by pmlandwehr** [comment]
(ping @ocefpaf for merge)

**Pull request merged**